controllers/operator_controller_test.go

```diff
@@ -348,180 +348,210 @@ var _ = Describe("Reconcile Test", func() {
 				Expect(err).NotTo(HaveOccurred())
 			})
 
-			It("verify operator status when bundle deployment is waiting to be created", func() {
```

Comment:
Perhaps as a follow-up. We have a `Describe("the expected bundle deployment already exists")` block at line 123. It seems like all of the tests in the `When("the existing operator status is based on bundleDeployment")` block at line 307 belong inside that block, right? That might reduce the LoC necessary to setup the tests since all of these status mapping tests assume the BD spec is already correct.

WDYT?

Reply:
Made the change and removed a duplicate test